manifests/complete-postgres-manifest.yaml

```diff
@@ -32,6 +32,8 @@ spec:
 #  spiloFSGroup: 103
 #  podAnnotations:
 #    annotation.key: value
+  serviceAnnotations:
```

Comment:
please use comments here, like we did for `podAnnotations`

Reply:
I use this value on the e2e test. I'll move this to a new manifest specific for `serviceAnnotations` and comment it.

Reply:
ah I see. Another approach could be to patch a running `postgresql` and see if the `serviceAnnotations` appear on the services

Reply:
indeed I like your solution but it'd not work since [`sameService`](https://github.com/zalando/postgres-operator/blob/master/pkg/util/k8sutil/k8sutil.go#L122) is not prepared to update service with annotations change. That's a known problem and will work on it in a separate PR soon. I will also add an e2e test for it as you described.